packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -29,6 +29,7 @@ class BuildIOSCommand extends BuildSubCommand {
     argParser.addFlag('simulator', help: 'Build for the iOS simulator instead of the device.');
     argParser.addFlag('codesign', negatable: true, defaultsTo: true,
         help: 'Codesign the application bundle (only available on device builds).');
+    argParser.addFlag('preview-dart-2', negatable: false);
```

Comment:
should these be hidden flags, btw? most of our users will never use this and it might be confusing to them.

Reply:
I don't have opinion on this. You are suggesting to make all `preview-dart-2` options `hide: !verboseHelp` everywhere, did I get it right?

Reply:
Right.

Reply:
Okay, it is actually already hidden for `run` command. https://github.com/flutter/flutter/pull/13270 hides it everywhere else.